src/steps/dataproc/index.ts

```diff
@@ -20,6 +26,21 @@ export async function fetchDataprocClusters(
   await client.iterateClusters(async (cluster) => {
     const clusterEntity = createDataprocClusterEntity(cluster);
     await jobState.addEntity(clusterEntity);
+
+    const kmsKey = cluster.config?.encryptionConfig?.gcePdKmsKeyName;
+    if (kmsKey) {
+      const kmsKeyEntity = await jobState.findEntity(kmsKey);
+
+      if (kmsKeyEntity) {
+        await jobState.addRelationship(
+          createDirectRelationship({
+            _class: RelationshipClass.USES,
+            from: clusterEntity,
+            to: kmsKeyEntity,
+          }),
+        );
+      }
+    }
```

Comment:
Could you create one example (using Dashboard/Terraform - up to you, whatever is easier) that uses KMS?

That way we'd have some real KMS data in the snapshots and could see/test that relationships are indeed getting built.

Another thing - could you use `getKmsGraphObjectKeyFromKmsKeyName()`? This seems to be getting used more and more in the newer code. Here's an example: https://github.com/JupiterOne/graph-google-cloud/blob/main/src/steps/big-query/index.ts#L64

Reply:
P.S Also if you could fix up the tests, seems something went wrong when I synced the changes from the target branch/master.